### ISSUE TITLE
2022-09-16 Patch release(s) plan (2.4.x/2.3.x)

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1883]
## 2.4.3 Release
- #1857 
- #1843
- ~#1795~
- #1800
- #1882 
- #1865 
- #1859 
- #1892 
- #1895
- #1889 
- #1905 

## 2.3.10 Release
- #1857 
- #1843
- ~#1795~
- #1800
- #1882
- #1865 
- #1859 
- #1892
- #1895 
- #1889

**Comment by joeparsons** [comment]
## Pick list
```
git cherry-pick 7f7f7a51a1b75d1a61c184d5d82dc466b13656fd
git cherry-pick b66078b854d004dcaa2a6fbd980c8edbb24eb415
git cherry-pick 331184bd5ec04e04439c91a46330311796ecef67
git cherry-pick 93c426e5e13180ea3b95e9882c29ca684b4c0dae
git cherry-pick e67e58eb12a24b2401de13490ad3151e60a29cc2
git cherry-pick 2a157c7e5f3322c8b0879d58627edad8f2c1dc7b
git cherry-pick 269fb80a80a557b06ab770f4d8abfbae3fc4e4e3
git cherry-pick 89e7c6f50ad4b4556df57112d1f460fb23a42909
git cherry-pick 2f0bb91bd3e00f14fd33dadd5fb644f2b4526edf
```

**Comment by joeparsons** [comment]
Discussed #1795 with @tadean and decided that improvement should be minor release only.

**Comment by joeparsons** [comment]
## Release notes snippet

### 2.3.10

`git log 2.3.9..HEAD --pretty='format:- %s'`
```
- Closes #1888: Apply upstream config schema changes to easy_breadcrumb.settings.yml. (#1889)
- Closes #937: Fix missing icons in accordion previews. (#1895)
- Closes #1891: Use privileged GH token for 'Create Release' workflow. (#1892)
- Closes #1858: Fix broken ProboCI 'Check for PHP errors and notices' step. (#1859)
- Closes #692: Add version to info.yml during release action. (#1865)
- Closes #1881: Fix broken Dependency check job in security.yml GH actions workflow. (#1882)
- Closes #1674 Fix up links in migrations (#1800)
- Closes #1534 Adding multi-byte character support to EntityEmbedProcess migrate plugin. (#1843)
- Closes #1856: Fix deprecation warning caused by empty card/accordion body fields on PHP 8.1+. (#1857)
```

### 2.4.3

`git log 2.4.2..HEAD --pretty='format:- %s'`
```
- Closes #1888: Apply upstream config schema changes to easy_breadcrumb.settings.yml. (#1889)
- Closes #937: Fix missing icons in accordion previews. (#1895)
- Closes #1891: Use privileged GH token for 'Create Release' workflow. (#1892)
- Closes #1858: Fix broken ProboCI 'Check for PHP errors and notices' step. (#1859)
- Closes #692: Add version to info.yml during release action. (#1865)
- Closes #1881: Fix broken Dependency check job in security.yml GH actions workflow. (#1882)
- Closes #1674 Fix up links in migrations (#1800)
- Closes #1534 Adding multi-byte character support to EntityEmbedProcess migrate plugin. (#1843)
- Closes #1856: Fix deprecation warning caused by empty card/accordion body fields on PHP 8.1+. (#1857)
```
